### PR TITLE
Us#93 detete subscription

### PR DESCRIPTION
**Pull request by apassett**

### CODE REVIEW

File: lib/metabase/endpoint/pulse.rb [hunk 1]
```diff
@@ -11,6 +11,15 @@ module Pulse
       def pulses(**params)
         get('/api/pulse', **params)
       end
+
+      # Delete a subscription.
+      #
+      # @param params [Hash] Query string
+      # @return Parsed response JSON
+      # @see https://github.com/metabase/metabase/blob/master/docs/api-documentation.md#get-apipulse
```
Comment (grustia01) [inline lib/metabase/endpoint/pulse.rb]: Delete subscription document url is https://github.com/metabase/metabase/blob/master/docs/api/pulse.md#delete-apipulseidsubscription

**Review by mrgutie8 — CHANGES_REQUESTED** [review]
see https://github.com/mrgutie8/metabase-ruby-rest-api/pull/62/files#r1404002971

**Review by mrgutie8 — APPROVED** [review]
lgtm

**Comment by apassett** [comment]
Had to resolve conflicts with the dev branch please re-review

**Pull request merged**